common/protob/messages-cardano.proto

```diff
@@ -15,6 +15,12 @@ enum CardanoAddressType {
     // todo: add other type
 }
 
+message CardanoStakingKeyPointer {
```

Comment:
Perhaps it should be named CardanoStakingKeyPointerType as with CardanoTxOutputType or CardanoTxInputType - since it is not a message.

Reply:
Makes sense to me. Also, I guess we can make its components required. If it is ever modified, we can replace the whole type.